src/components/showcase-settings.component.tsx

```diff
@@ -20,32 +22,35 @@ export interface ShowcaseSettingsProps extends ViewProps {
   onReset: () => void;
 }
 
-export const ShowcaseSettings = (props: ShowcaseSettingsProps): React.ReactElement => {
-
-  const [themesMenuVisible, setThemesMenuVisible] = React.useState<boolean>(false);
-  const [settingsMenuVisible, setSettingsMenuVisible] = React.useState<boolean>(false);
+export const ShowcaseSettings = (
+  props: ShowcaseSettingsProps,
+): React.ReactElement => {
+  const [themesMenuVisible, setThemesMenuVisible] = React.useState<boolean>(
+    false,
+  );
+  const [settingsMenuVisible, setSettingsMenuVisible] = React.useState<boolean>(
+    false,
+  );
 
-  const createSettingMenuItem = (setting: ComponentShowcaseSetting): OverflowMenuItemType => {
-    return {
-      title: setting.description || `${setting.propertyName}: ${setting.value}`,
-    };
+  const createSettingMenuItem = (setting: ComponentShowcaseSetting, index: number): ReactElement => {
+    return <MenuItem key={index} title={setting.description || `${setting.propertyName}: ${setting.value}`} />;
   };
 
-  const createThemeMenuItem = (title: string): OverflowMenuItemType => {
-    return { title };
+  const createThemeMenuItem = (title: string, index: number): ReactElement => {
+    return <MenuItem key={index} title={title} />;
   };
```

Comment:
Let's follow the existing code style and:
- wrap the returned component into the round braces
- use one line per prop

```suggestion
  const createThemeMenuItem = (title: string, index: number): React.ReactElement => (
    <MenuItem
      key={index}
      title={title}
    />
  );
```